File: AppApi/AppApi.Entities/DTO/Shoe-receive/SaveReceiveShoeDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppApi.Entities.DTO.Shoe_receive
{
    public class SaveReceiveShoeDto
    {
        public int ShoeOrderId { get; set; }
        public string ReceiveUser { get; set; }
        public string ReceiveNo { get; set; }
        public string OrderNo { get; set; }
        public DateTime ReceiveDate { get; set; }
        public List<ShoeReceiveDetailDto> ShoesList { get; set; }
        public int CheckShoeOrderComplete { get; set; } // kiểm tra đơn hàng đã nhận được hết hay chưa

    }
}

File: angular/src/app/_models/shoe-receive/SaveReceiveShoeDto.ts
import { Moment } from "moment";
import { ShoeReceiveDetailDto } from "./ShoeReceiveDetailDto";

export class SaveReceiveShoeDto {
    ShoeOrderId: number;
    ReceiveUser: string;
    ReceiveNo:string;
    OrderNo:string;
    ReceiveDate: Moment | null;
    ShoesList: ShoeReceiveDetailDto[];
    CheckShoeOrderComplete: number; // kiểm tra đơn hàng đã nhận được hết hay chưa
  }
